### PR TITLE
#get token param removed, added helper methods, changed version param…

### PR DESCRIPTION
**Pull request by chandeeland**
… to query param

### CODE REVIEW

File: lib/frodo/profile.rb [hunk 1]
```diff
@@ -2,16 +2,32 @@
 
 module Frodo
   class Profile
-    def self.get(id:, token:, version: 'latest')
-      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}/#{version}"
-      new(gandalf_url: gandalf_url, token: token)
+    def self.get(id:, version: 'latest')
+      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}?version=#{version}"
```
Comment (alexshook) [inline lib/frodo/profile.rb]: Is there ever a chance that our environment variable will not be a string? Is to protect against nils?

Comment (chandeeland) [reply lib/frodo/profile.rb]: yeah if nil you get undefined method +

File: spec/frodo/profile_spec.rb [hunk 2]
```diff
@@ -156,4 +155,11 @@
       end
     end
   end
+
+  describe described_class.new(gandalf_url: 'test') do
```
Comment (aperezmontan) [inline spec/frodo/profile_spec.rb]: Is this test right ?

File: lib/frodo/profile.rb [hunk 3]
```diff
@@ -2,16 +2,32 @@
...
+    def version
+      data.dig('data', 'attributes', 'version')
+    end
+
+    def configs
+      data.dig('data', 'attributes', 'data', 'configs')
+    end
+
+    def client_applications
+      data.dig('data', 'attributes', 'data', 'configs')
```
Comment (aperezmontan) [inline lib/frodo/profile.rb]: Nah bruh.

Tests would've caught this btw

**Review by inactdev — CHANGES_REQUESTED** [review]

File: spec/frodo/profile_spec.rb [hunk 4]
```diff
@@ -16,12 +16,11 @@
   context '.get' do
     before do
       ENV['GANDALF_URL'] = 'https://www.gandalf.com'
```
Comment (alexshook) [inline spec/frodo/profile_spec.rb]: Do we have `wrap_env` here in this repo?

**Review by alexshook — APPROVED** [review]

File: lib/frodo/profile.rb (hunk 3 above)
Comment (alexshook) [reply lib/frodo/profile.rb]: Oh I approved but yeah this is a duplicate, this traversal is wrong.

**Pull request merged**